client/src/App.js

```diff
@@ -26,6 +27,7 @@ function App() {
           <Route path='/create-contest' component={CreateContestPage}></Route>
           <Route path='/submit-design' component={SubmitDesignPage}></Route>
           <Route path='/' component={LandingPage} />
+          <Route path="/add_card" component={PaymentInfo} />
```

Comment:
```suggestion
          <Route path="/add-card" component={PaymentInfo} />
```

routes should generally be camelCase or separated by hyphens. we can reserve snake_case for the backend variables